int8 STATE_INITIALIZE=0
int8 STATE_EXPLORE=1
int8 STATE_RELAY=2
int8 STATE_SACRIFICE=3
int8 STATE_GOHOME=4

int8 state
---
bool success
